landbosse/excelio/XlsxParallelManagerRunner.py

```diff
@@ -69,12 +69,14 @@ def run_from_project_list_xlsx(self, projects_xlsx, log):
         # Get the output dictionary ready
         runs_dict = {project_id: result for project_id, result in executor_result}
 
-        # .csv lists for all runs
-        csv_lists = self.extract_csv_lists(runs_dict)
-        module_type_operation_lists = self.extract_module_type_operation_lists(runs_dict)
+        # Assemble the dictionary with content for the details, details with inputs,
+        #  cost_by_module_type_operation and cost_by_module_type_operation_with_input tabs
+        final_result = dict()
+        final_result['details_list'] = self.extract_details_lists(runs_dict)
```

Comment:
I switched to a dictionary here because, if we need to add more tabs for scaling analysis, it is easier to keep track of key names in a dictionary than a long list of elements in tuples.